src/styles/index.scss

```diff
@@ -1,3 +1,63 @@
 body {
+  display: flex;
   margin: 0;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+$day-size: 100px;
+$border-size: 1px;
+$padding: 10px;
+$hover-color: #ffbfcb;
+$day-color: #eee;
+$hover-transform: -20px;
+$transition-duration: 0.5s;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: calc(7 * #{$day-size} + 7px);
```

Comment:
The calculation for the calendar width is correct, but ensure that the gap between days is consistent with the `gap` property set on the `.calendar` class.